### PR TITLE
Update tpm2_util.h

### PR DESCRIPTION
**Pull request by yunusemreayhan**
Not compiling on arm-unknown-linux-uclibcgnueabi-gcc

### CODE REVIEW

File: tools/tpm2_pcrlist.c [hunk 1]
```diff
@@ -253,7 +253,7 @@ static bool show_pcr_values(void) {
 
         tpm2_tool_output("%s :\n", alg_name);
 
-        UINT32 pcr_id;
+        INT32 pcr_id;
```
Comment (williamcroberts) [inline tools/tpm2_pcrlist.c]: You need to drop this commit.

**Comment by yunusemreayhan** [comment]
I have updated pr, please squash into one commit, and merge.
thx.

**Comment by williamcroberts** [comment]
@yunusemreayhan you didn't update it correctly. You now have 3 commits on this PR. All you needed to do was do a git rebase -i origin/master and drop the commit that shouldn't have been on this branch. Instead, you added another commit that was already merged.

I can't do much on the web-interface so its not as simple as just "squashing it down" because you have merge conflicts.

Normally I wouldn't fix this up manually and keep making you attempt to do it, so you know how to do it. But ill do it, this one time.

**Comment by williamcroberts** [comment]
Moved: https://github.com/intel/tpm2-tools/pull/754

**Pull request closed without merging**